IdentityCore/src/requests/broker/MSIDSSOExtensionInteractiveTokenRequest.m

```diff
@@ -64,9 +65,18 @@ - (instancetype)initWithRequestParameters:(MSIDInteractiveRequestParameters *)pa
         _extensionDelegate = [MSIDSSOExtensionTokenRequestDelegate new];
         _extensionDelegate.context = parameters;
         __weak typeof(self) weakSelf = self;
-        _extensionDelegate.completionBlock = ^(MSIDTokenResponse *response, NSError *error)
+        _extensionDelegate.completionBlock = ^(MSIDBrokerOperationTokenResponse *operationResponse, NSError *error)
         {
-            [weakSelf handleTokenResponse:response error:error completionBlock:weakSelf.requestCompletionBlock];
+            MSIDInteractiveRequestCompletionBlock blockWrapper = ^(MSIDTokenResult *result, NSError *error, MSIDWebWPJResponse *installBrokerResponse)
+            {
+                NSMutableDictionary *userInfo = error.userInfo ? [error.userInfo mutableCopy] : [NSMutableDictionary new];
+                userInfo[MSIDBrokerVersionKey] = operationResponse.clientAppVersion;
```

Comment:
can this be passed in the error directly to avoid changing it here?

Reply:
Added app version to token response.